lightning/src/ln/peer_handler.rs

```diff
@@ -1309,18 +1285,14 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, L: Deref> PeerManager<D
 							msg.first_blocknum,
 							msg.number_of_blocks,
 							msg.sync_complete);
-						let (mut descriptor, peer) = get_peer_for_forwarding!(node_id);
+						let (_, peer) = get_peer_for_forwarding!(node_id);
```

Comment:
I think the first tuple item of `get_peer_for_forwarding`'s result is never used anymore, could be dropped